<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSequenceDialog</class>
 <widget class="QDialog" name="EditSequenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="globalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Annotations region resolving mode</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QRadioButton" name="resizeRB">
          <property name="text">
           <string>Expand affected annotation</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="removeRB">
          <property name="text">
           <string>Remove affected annotation</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="splitRB">
          <property name="text">
           <string>Split (join annotation parts)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="split_separateRB">
          <property name="text">
           <string>Split (separate annotations parts)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="topMargin">
           <number>7</number>
          </property>
          <item>
           <widget class="QCheckBox" name="recalculateQualsCheckBox">
            <property name="text">
             <string>Recalculate values of qualifiers</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="insertPositionBox">
       <property name="title">
        <string>Position to insert</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QToolButton" name="startPosToolButton">
          <property name="toolTip">
           <string>Press to insert before the start of the sequence</string>
          </property>
          <property name="text">
           <string>start</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="insertPositionSpin">
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="endPosToolButton">
          <property name="toolTip">
           <string>Press to insert to the end of the sequence</string>
          </property>
          <property name="text">
           <string>end</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QGroupBox" name="selectionGroupBox">
          <property name="title">
           <string>Selection</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QToolButton" name="beforeSelectionToolButton">
             <property name="toolTip">
              <string>Press to insert before the selection</string>
             </property>
             <property name="text">
              <string>before</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="afterSelectionToolButton">
             <property name="toolTip">
              <string>Press to insert after the selection</string>
             </property>
             <property name="text">
              <string>after</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="selectionLineEdit">
             <property name="toolTip">
              <string>To change the regsion select it on a sequence</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="saveToAnotherBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Save to new file</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Document location:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="filepathEdit"/>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="browseButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Document format:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="formatBox"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="mergeAnnotationsBox">
        <property name="text">
         <string>Merge annotations to this file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditSequenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditSequenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
